### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by karayamanemre**
- Created the React App.
- Connected React and Redux.
- Wrote integration tests with a React testing library.
- Applied React best practices and language style guides in code.

### CODE REVIEW

File: src/Components/Home.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            <h1>Loading...</h1>
+            {' '}
+          </div>
+        )}
+      </div>
+    </div>
+  );
+};
+
+export default Home;
```
Comment (TedLivist) [inline src/Components/Home.js:49]: - [x] Kindly implement the alternating color patterns as we have in the design. As you can see the darker and lighter colors are alternating.

  Your Implementation | Behance Design
  :---:|:---:
  ufi | jonathas-orig-home

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
```
Comment (TedLivist) [inline README.md]: - [x] Towards the end of your README there should be an "Acknowledgement" section that includes credit to Nelson Sakwa whose design was used.

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
-# crypto-market-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Crypto Market App
+
+> Crypto Market is a SPA that allow users to check live prices of cryptocurrencies. Built with React and CoinGecko API.
```
Comment (TedLivist) [inline README.md:5]: - [x] Kindly deploy your app and include the link in your README.

- [x] Kindly record a video presentation about the project no longer than 5 minutes and provide the link. Remember also to highlight the piece of code you are proud of making.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @karayamanemre,

Good job so far!
✔️ Good documentation
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @karayamanemre  👋 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

img

## Highlights
- Descriptive PR 👍 
- Descriptive Readme file ✔️ 
- No linter errors 💯 

## Suggested changes
_Check the comments under the review._

- [x] Please, As suggested by the previous reviewer, try making your presentation more descriptive and remember to highlight the piece of code you are most proud of. you got this 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ShadyShawkat — APPROVED** [review]
## Hi @karayamanemre, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**